### PR TITLE
cget package

### PR DESCRIPTION
**Pull request by ilbiondo**
New package for cget

### CODE REVIEW

File: var/spack/repos/builtin/packages/cget/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Cget(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cget/package.py:9]: Rename to py-cget

File: var/spack/repos/builtin/packages/cget/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    """Cmake package retrieval."""
+
+    homepage = "https://github.com/pfultz2/cget"
+    url      = "https://pypi.io/packages/source/c/cget/cget-0.1.9.tar.gz"
+
+    version('0.1.9', sha256='2a7913b601bec615208585eda7e69998a43cc17080d36c2ff2ce8742c9794bf6')
+
+    depends_on("py-setuptools")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cget/package.py]:
```suggestion
    depends_on("py-setuptools", type='build')
```

File: var/spack/repos/builtin/packages/cget/package.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    depends_on("py-six")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cget/package.py]:
```suggestion
    depends_on("py-six@1.10:", type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/cget/package.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    depends_on("py-click")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cget/package.py]:
```suggestion
    depends_on("py-click@6.6:", type=('build', 'run'))
    depends_on('py-subprocess32', when='^python@:2', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/cget/package.py (hunk 1 above)
Comment (ilbiondo) [reply var/spack/repos/builtin/packages/cget/package.py:9]: As per the other PRs it's a stand alone tool - one calls `cget` on the command line and never `import * from cget`

https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name

File: var/spack/repos/builtin/packages/cget/package.py (hunk 2 above)
Comment (ilbiondo) [reply var/spack/repos/builtin/packages/cget/package.py]: Correct

File: var/spack/repos/builtin/packages/cget/package.py (hunk 3 above)
Comment (ilbiondo) [reply var/spack/repos/builtin/packages/cget/package.py]: Looks good

File: var/spack/repos/builtin/packages/cget/package.py (hunk 4 above)
Comment (ilbiondo) [reply var/spack/repos/builtin/packages/cget/package.py]: Ah yes, python2 is still clinging on out there....

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**